aerospike-mixin/dashboards/aerospike-overview.libsonnet

```diff
@@ -0,0 +1,1176 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'aerospike-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local nodesPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(aerospike_cluster, instance) (aerospike_namespace_ns_cluster_size{job=~"$job", aerospike_cluster=~"$aerospike_cluster"})',
```

Comment:
this doesn't seem to take into account whether the `cluster` selector should be used. I'm guessing this applies to all panels and all dashboards. take a look at https://github.com/grafana/jsonnet-libs/blob/62aec8403a5c38d5dc97ba596703753289b1c33b/apache-cassandra-mixin/dashboards/cassandra-overview.libsonnet#L1781 for an example